templates/components/dateRange.build.hbs

```diff
@@ -0,0 +1,46 @@
+<template>
+  <label
+    v-if="showPredefinedRanges"
+    :for="name"
+    class="select-proxy-display"
+    :class="{ 'input-focused': isInputFocused , 'readonly': readonly }"
+  >
+    <select
+      class="form-control hidden-select focus-outline"
+      name="predefinedRangesOptions"
+      id="predefinedRangesOptions"
+      v-model.lazy="selectedRange"
+      v-on:keydown.prevent="readonly ? false : true"
+      tabindex="0"
+      @blur="onBlur()"
+    >
+      <option v-if="placeholder" value="">\{{ placeholder }}</option>
```

Comment:
If predefined options need to be shown, then this option should always be available. It shouldn't have anything to do with whether there's a truthy value for `placeholder`.